tests/integration/targets/autoscaling_instance_refresh/tasks/instance_refresh_info.yml

```diff
@@ -0,0 +1,101 @@
+---
+- name: Test getting info for an ASG name
+  autoscaling_instance_refresh_info:
+    name: "{{ asg_name }}"
+  register: output
+
+- name: Assert that the correct number of records are returned
+  assert:
+    that:
+      - output | community.general.json_query(inst_refresh_id_json_query) | unique | length == 7
```

Comment:
Can you please replace `community.general.json_query` and use the json filter instead? We may want to limit dependencies on other collections.